cli.js

```diff
@@ -47,7 +47,7 @@ if (argv.c) {
 }
 
 function validateCmdVariable (param) {
-  if (!param.match(/^\w+=\w+$/)) {
+  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&-/:;.]+$/)) {
```

Comment:
```suggestion
  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&\-/:;.]+$/)) {
```
The `=` might cause problems when not within `"` but 🤷 

You missed an escape before the `-` literal so I suggest to add that